test-files/cases/hoot-js/line-generic-1/Expected.osm

```diff
@@ -1,128 +1,129 @@
+    <node visible="true" id="-27581" timestamp="1970-01-01T00:00:00Z" version="1" lat="38.8535732961699978" lon="-104.8997265932799934"/>
+    <node visible="true" id="-27580" timestamp="1970-01-01T00:00:00Z" version="1" lat="38.8538862746800078" lon="-104.8997029521899833"/>
+    <node visible="true" id="-27579" timestamp="1970-01-01T00:00:00Z" version="1" lat="38.8542508008399921" lon="-104.8997596908100007"/>
+    <node visible="true" id="-27578" timestamp="1970-01-01T00:00:00Z" version="1" lat="38.8543116507600033" lon="-104.8997596908100007"/>
+    <node visible="true" id="-27577" timestamp="1970-01-01T00:00:00Z" version="1" lat="38.8549467092499938" lon="-104.8997596908100007"/>
+    <node visible="true" id="-27576" timestamp="1970-01-01T00:00:00Z" version="1" lat="38.8549614373999930" lon="-104.8999833093500058"/>
+    <node visible="true" id="-27575" timestamp="1970-01-01T00:00:00Z" version="1" lat="38.8549596336300098" lon="-104.9003909734099977"/>
+    <node visible="true" id="-27574" timestamp="1970-01-01T00:00:00Z" version="1" lat="38.8549470072699989" lon="-104.9004674104200063"/>
+    <node visible="true" id="-27573" timestamp="1970-01-01T00:00:00Z" version="1" lat="38.8549307733599960" lon="-104.9005623776100009"/>
+    <node visible="true" id="-27572" timestamp="1970-01-01T00:00:00Z" version="1" lat="38.8548694452500030" lon="-104.9007059865400038"/>
+    <node visible="true" id="-27571" timestamp="1970-01-01T00:00:00Z" version="1" lat="38.8547540039599895" lon="-104.9009352975700011"/>
+    <node visible="true" id="-27570" timestamp="1970-01-01T00:00:00Z" version="1" lat="38.8544527734400020" lon="-104.9015560587500033"/>
+    <node visible="true" id="-27569" timestamp="1970-01-01T00:00:00Z" version="1" lat="38.8543571730699995" lon="-104.9017344117799837"/>
+    <node visible="true" id="-27568" timestamp="1970-01-01T00:00:00Z" version="1" lat="38.8542940406799957" lon="-104.9018896020699998"/>
+    <node visible="true" id="-27567" timestamp="1970-01-01T00:00:00Z" version="1" lat="38.8542146741700023" lon="-104.9020934340999958"/>
+    <node visible="true" id="-27566" timestamp="1970-01-01T00:00:00Z" version="1" lat="38.8541136621199996" lon="-104.9023412753199977"/>
+    <node visible="true" id="-27565" timestamp="1970-01-01T00:00:00Z" version="1" lat="38.8540360991899973" lon="-104.9024200286000053"/>
+    <way visible="true" id="-27591" timestamp="1970-01-01T00:00:00Z" version="1">
+        <nd ref="-27565"/>
+        <nd ref="-27566"/>
+        <nd ref="-27567"/>
+        <nd ref="-27568"/>
+        <nd ref="-27569"/>
+        <nd ref="-27570"/>
+        <nd ref="-27571"/>
+        <nd ref="-27572"/>
+        <nd ref="-27573"/>
+        <nd ref="-27574"/>
+        <nd ref="-27575"/>
+        <nd ref="-27576"/>
+        <nd ref="-27577"/>
+        <nd ref="-27578"/>
+        <nd ref="-27579"/>
+        <nd ref="-27580"/>
+        <nd ref="-27581"/>
+        <nd ref="-27582"/>
```

Comment:
Not only are there changes in IDs, the results also have nodes out of order because we now sort by ID in the output.  Some files are affected more than others.